o.n.bootstrap/launcher/windows/jvmlauncher.cpp

```diff
@@ -405,6 +407,13 @@ bool JvmLauncher::findJava(const char *minJavaVersion) {
     if (findJava(JRE_KEY, "", minJavaVersion)) {
         return true;
     }
+// scan for registry for jdk/jre version 9
+    if (findJava(JDK_POST9_KEY, "", minJavaVersion)) {
+        return true;
+    }
+    if (findJava(JRE_POST9_KEY, "", minJavaVersion)) {
+        return true;
+    }
```

Comment:
Wouldn't it be better to check the new layout first to start the new version in favor of the old one?

Reply:
Adjusted to take more recent JDK first